util/util.go

```diff
@@ -502,6 +502,21 @@ func MoveDir(srcDir string, destDir string) error {
 	return nil
 }
 
+func CallInDir(path string, execFunc func() error) error {
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	os.Chdir(path)
```

Comment:
It is probably a good idea to check for errors here.

Also, this is just a stylistic comment - I think it is more "Go like" to switch back to the original directory using `defer`.  Then you can continue to return on error.